fast/stages/0-bootstrap/organization.tf

```diff
@@ -45,6 +45,11 @@ locals {
       }
     ]
   ])
+  drs_domains = concat(
+    [var.organization.customer_id],
+    var.org_policies_config.constraints.allowed_policy_member_domains
```

Comment:
`var.org_policies_config` can be null. Either check for null values or (better) make the variable non-nullable?